config/yaml.go

```diff
@@ -314,14 +334,9 @@ func unmarshalYAMLValue(reader io.ReadCloser, value interface{}, lookUp lookUpFu
 // will be used
 //
 // TODO: what if someone wanted a literal ${FOO} in config? need a small escape hatch
```

Comment:
Thanks, I've forgot to update the changelog.
Yes, syntax is applicable only in values, in order to escape these values, we'll need to use something other than os.Expand.